pom.xml

```diff
@@ -311,6 +312,16 @@
         <artifactId>testng</artifactId>
         <version>6.11</version>
       </dependency>
+      <dependency>
```

Comment:
Mockito is not good enough for the tests? To add this, maybe we need to check its license

Reply:
No, Mockito cannot mock static object. In this case, Pinot controller contains more than 1 singleton object and we need to instantiate more than 1 pinot controller in our test.

Reply:
Removed PowerMock.